### PR TITLE
feat: enable new model info in preditct page

### PR DESCRIPTION
**Pull request by alex-krasn**
on issue #377

### CODE REVIEW

File: src/react/components/pages/predict/predictModelInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import React from 'react'
+import './predictModelInfo.scss';
+
+export default function PredictModelInfo({ modelInfo }) {
+    const { docType, modelId, docTypeConfidence } = modelInfo;
+    const getPercentage = (value: number) => {
+        const percents = 100 * value;
+        return percents % 1 !== 0 ? percents.toFixed(2) : percents;
+    };
+    const modeType = docType.split(":")[0]
```
Comment (stew-ro) [inline src/react/components/pages/predict/predictModelInfo.tsx:13]: missing semi colons

File: src/react/components/pages/predict/predictModelInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    return (
+        <div className="model-info-container">
+            <div className="model-info-item">
+                <span className="title" >Model type:</span>
+                <span className="value" >{modeType}</span>
+            </div>
+            {
+                modelName !== modelId &&
+            <div className="model-info-item">
```
Comment (stew-ro) [inline src/react/components/pages/predict/predictModelInfo.tsx:24]: indent

File: src/react/components/pages/predict/predictModelInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                <span className="title" >Model name:</span>
+                <span className="value" >{modelName}</span>
+            </div>
```
Comment (stew-ro) [inline src/react/components/pages/predict/predictModelInfo.tsx:27]: indent

**Comment by alex-krasn** [comment]
closes #377

**Pull request merged**